### PR TITLE
README for Chinese

### PR DESCRIPTION
**Pull request by yosukehigashi**

### CODE REVIEW

**Comment by yosukehigashi** [comment]
@anelephantbowl PTAL when you can! This is all GPT4 translated so I have no idea what anything says 😅

File: README_zh.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+```python
+# 选择一些指标
+sentiment_values = langcheck.metrics.zh.sentiment(generated_outputs)
+toxicity_values = langcheck.metrics.zh.toxicity(generated_outputs)
+
+# 关于一个指标的交互式散点图
+sentiment_values.scatter()
+```
+
+![Scatter plot for one metric](docs/_static/scatter_one_metric_ja.gif)
```
Comment (yosukehigashi) [inline README_zh.md:81]: @anelephantbowl Could you suggest some phrases that we can use for this demo? A few extra variations of these kinds of phrases would be helpful
```
generated_outputs = [
    '很高兴能够帮到您。',
    '我很高兴回答您的问题。',
    '我不太清楚，请自己查找。'
]
```

Also, if you can tell me what would be a reasonable word to filter for (as we do in the Japanese gif) that would be helpful!

**Comment by anelephantbowl** [comment]
I will take a look later this week!

**Comment by Vela-zz** [comment]
I can also help in add/fix content in chinese readme.

**Comment by anelephantbowl** [comment]
@Vela-zz That would be helpful too! Thanks for offering!

**Comment by yosukehigashi** [comment]
> I can also help in add/fix content in chinese readme.

Thank you for the edits! @anelephantbowl FYI @Vela-zz made some changes in https://github.com/citadel-ai/langcheck/pull/98

**Review by anelephantbowl — APPROVED** [review]
LGTM!

**Pull request merged**